### DISCUSSION TITLE
Add setting to make input history show by default

### DISCUSSION THREAD

**Discussion by Banzobotic** [monkeytypegame/monkeytype#1139] [Ideas] [answered]
I regularly want to look at my input history and it would be nice if there was a "Display input history by default" setting in appearance to save some time.

**Comment by Miodec** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/13181393/112387288-7ac60b00-8ce9-11eb-91fb-f6276ac5bae4.png)

**Reply by Banzobotic** [reply]
Thanks, I don't know how I missed that
